ios/Exponent/Versioned/Core/UniversalModules/EXScopedModuleRegistryAdapter.m

```diff
@@ -59,7 +59,16 @@ - (UMModuleRegistry *)moduleRegistryForParams:(NSDictionary *)params forExperien
 #endif
 
 #if __has_include(<EXFileSystem/EXFileSystem.h>)
-  EXScopedFileSystemModule *fileSystemModule = [[EXScopedFileSystemModule alloc] initWithExperienceId:experienceId andConstantsBinding:constantsBinding];
+  EXScopedFileSystemModule *fileSystemModule;
+  if (params[@"fileSystemDirectories"]) {
+    NSString *documentDirectory = params[@"fileSystemDirectories"][@"documentDirectory"];
+    NSString *cachesDirectory = params[@"fileSystemDirectories"][@"cachesDirectory"];
+    fileSystemModule = [[EXScopedFileSystemModule alloc] initWithDocumentDirectory:documentDirectory
```

Comment:
I'm wondering if it still has to be scoped 🤔
Maybe this initializer could be a part of the original module? Right now you no longer pass an `experienceId` nor constants binding here. Of course it's still scoped but in different way than other scoped modules requiring `experienceId` or having some expo-managed-specific code 🤔

Reply:
This initializer is a part of the original module iirc

Reply:
https://github.com/expo/expo/blob/c44b761c0671493da060889642db162125c166b3/packages/expo-file-system/ios/EXFileSystem/EXFileSystem.h#L15